layers/+distributions/spacemacs-base/funcs.el

```diff
@@ -184,6 +184,62 @@ Dedicated (locked) windows are left untouched."
   (interactive "p")
   (spacemacs/rotate-windows (* -1 count)))
 
+;; Note that this duplicates code from select-window-by-number, ideally should
+;; upstream this function into windows.el
+(defun spacemacs/get-window-by-number (i)
+  (let ((windows (car (gethash (selected-frame) window-numbering-table)))
```

Comment:
And it seems that this file already contains the same definition of `spacemacs/get-window-by-number`.